packages/fluentui/react-bindings/src/FocusZone/FocusZone.tsx

```diff
@@ -45,7 +45,25 @@ const _allInstances: {
   [key: string]: FocusZone;
 } = {};
 
-const _outerZones: Set<FocusZone> = new Set();
+const outZones = {
+  _outerZones: new Map<Window, Set<FocusZone>>(),
```

Comment:
nit on naming: rename `_outerZones` to `_windowToOuterZoneMap` or something

Reply:
Done